src/neo4j_graphrag/experimental/components/schema.py

```diff
@@ -19,12 +19,12 @@
 from pydantic import BaseModel, ValidationError, model_validator, validate_call
 
 from neo4j_graphrag.exceptions import SchemaValidationError
-from neo4j_graphrag.experimental.pipeline import Component, DataModel
+from neo4j_graphrag.experimental.pipeline.component import Component, DataModel
 
 
 class SchemaProperty(BaseModel):
     """
-    Represents a property on a node or relationship in the graph.
+    represents a property on a node or relationship in the graph.
```

Comment:
Not sure about that :)

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/165/commits/cf973bd6090afaa2f842a394bb4ed88ce90ceaa2